### PR TITLE
fix(GRA-998): fix peer rendering, update node model

### PR DESCRIPTION
**Pull request by Aceix**
TESTING STEPS:
1. Start netclient gui
2. Join a network with peers
3. Check that the peers show

### CODE REVIEW

**Comment by Aceix** [comment]
image

File: go.mod [hunk 1]
```diff
@@ -21,103 +21,95 @@ require (
 	github.com/hashicorp/go-version v1.6.0
 	github.com/matryer/is v1.4.0
 	github.com/vishvananda/netlink v1.1.0
-	github.com/wailsapp/wails/v2 v2.3.1
+	github.com/wailsapp/wails/v2 v2.2.0
```
Comment (0xdcarns) [inline go.mod:24]: any reason for a downgrade here?

**Review by 0xdcarns — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (Aceix) [reply go.mod]: rectified. had to update the wails cli so it doesnt auto downgrade

**Review by 0xdcarns — APPROVED** [review]

**Comment by theguy951357** [comment]
testing results in GRA-998

**Review by theguy951357 — APPROVED** [review]
can see peers on the gui and the infinite loop warning popup is gone 
pass

**Pull request merged**